c2rust-refactor/tests/remove_literal_suffixes_token_stream/old.rs

```diff
@@ -0,0 +1,15 @@
+// Regression input for the token-stream mismatch caused by editing vec! literals.
+
+fn main() {
+    let ints = vec![10u32, 20u32, 30u32];
```

Comment:
Are we adding, or deleting a token stream here?

Reply:
> Are you saying there could be some attribute that results in comparing two Some(_) values rather than (None, Some) such that a crash wouldn't have happened?

It shouldn't be doing the comparison in the first place, can you figure out where it's happening? We have `LazyTokenStream` set to ignore
```
#[equiv_mode=ignore] #[match=ignore] #[rewrite_ignore]
flag LazyTokenStream;
```
so the two streams should always compare as equal.